addons/udes_stock/models/stock_picking.py

```diff
@@ -267,6 +274,12 @@ def _compute_state(self):
 
         super()._compute_state()
 
+    @api.depends('state')
+    @api.one
```

Comment:
Q: Is there any chance this would error? Don't computed fields usually loop through a recordset in self?

Reply:
I was concerned that it might error, or recurse endlessly, so I used the same structure as the immediately preceding `_compute_state` method.